packages/hash/api/src/db/adapter.ts

```diff
@@ -157,6 +157,26 @@ export type Graph = {
   }[];
 };
 
+export type DbBlockProperties = {
+  entityId: string;
+  accountId: string;
+  componentId?: string;
+};
+
+export type DbPageProperties = {
+  archived?: boolean | null;
+  contents: DbBlockProperties[];
```

Comment:
I assume this might be the reason for the optional `componentId` above. 
I believe `componentId` is required for blocks. so could be okay to remove optionality above.
Perhaps @CiaranMn has some insights about this